manager/controllers/system_packages.go

```diff
@@ -47,6 +47,8 @@ var SystemPackagesOpts = ListOpts{
 type SystemPackageDBLoad struct {
 	SystemPackagesAttrs
 	Updates []byte `json:"updates" query:"spkg.update_data" gorm:"column:updates"`
+	// a helper to get total number of systems
+	Total int `json:"-" csv:"-" query:"count(*) over ()" gorm:"column:total"`
```

Comment:
`Total` is not used in query because the query is made with `SystemPackagesSelect`. It looks like it should be enough to change line 36 from
```go
var SystemPackagesSelect = fmt.Sprintf("%s,spkg.update_data as updates", database.MustGetSelect(&SystemPackagesAttrs{}))
```
to
```go
var SystemPackagesSelect = database.MustGetSelect(&SystemPackageDBLoad{})
```

Reply:
Thanks.